libs/aml/src/amlParser.ts

```
@@ -62,53 +64,54 @@
 const DocMultiline = createToken({ name: 'DocMultiline', pattern: /\|\|\|[^]*?\|\|\|/, line_breaks: true })
-const Comment = createToken({ name: 'Comment', pattern: /#[^\n]*/ })
+const Expression = createToken({ name: 'Expression', pattern: /`[^`]+`/ })
+const Identifier = createToken({ name: 'Identifier', pattern: /\b[a-zA-Z_][a-zA-Z0-9_#]*\b|"([^\\"]|\\\\|\\"|\\n)*"/ })
+const NewLine = createToken({ name: 'NewLine', pattern: /\r?\n/ })
+const WhiteSpace = createToken({name: 'WhiteSpace', pattern: /[ \t]+/})
 
 // values
-const Null = createToken({ name: 'Null', pattern: /null/i })
 const Decimal = createToken({ name: 'Decimal', pattern: /\d+\.\d+/ })
 const Integer = createToken({ name: 'Integer', pattern: /\d+/, longer_alt: Decimal })
 const String = createToken({ name: 'String', pattern: /'([^\\']|\\\\|\\')*'/ })
```

Biome

[error] Do not shadow the global "String" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)